spec/features/users_rate_other_user_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature "user chats with other user" do
+  context "user signs in and goes to conversation" do
+    let(:conversation) { FactoryGirl.create(:conversation)}
+
+    before do
+      visit root_path
+      click_link "Sign In"
+      fill_in 'Email', with: conversation.sender.email
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.